support/module/packaged/modules.json

```diff
@@ -55,8 +55,8 @@
 	},
 	"hyperloop": {
 		"hyperloop": {
-			"url": "https://github.com/appcelerator/hyperloop.next/releases/download/v7.0.4/hyperloop-7.0.4.zip",
-			"integrity": "sha512-5Nu6+o5LfE96wp5KHUR8Lj2iyDO3GrOk3WI5jO/RpD3pTxM9Yxuq2oyvfB4xXu/HnDYFsUKNRqIxsVU+9/AAJA=="
-		}
+      "url": "https://github.com/tidev/hyperloop.next/releases/download/v7.0.4/hyperloop-7.0.4.zip",
```

Comment:
tabs are missing (same for the three lines below

Reply:
That’s vscode formatting. Is that critical?

Reply:
either use tabs or spaces but not both in one file. Here in that file you've mixed both styles. 
And you can always disable "format on save" option if it makes problems. Or set it to use the files default (at least there was an option for Atom to check and use the format the file uses)